CHANGELOG.md

```diff
@@ -2,23 +2,23 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
-3.0.0-preview.1
-===============
+3.0.0-preview.1 (July 25, 2022)
+===============================
 
 New Features
 ------------
 
 **Large Rooms Pilot**
 
-twilio-video.js now allows you to create and join a [Large Room](TODO_doc_link), which supports a large number of Participants (> 50).
-**Note that the following API changes apply only for Large Rooms, and not for non-large group Rooms and Peer-to-Peer Rooms.**
+twilio-video.js now allows you to create and join a [Large Group Room](https://www.twilio.com/docs/video/large-group-rooms),
+which supports a large number of Participants (> 50). **Note that the following API changes apply only for Large Rooms,
```

Comment:
Should we add a line like "This feature is in pilot and requires users to have their account enabled to use it."